src/main/kotlin/org/jitsi/nlj/transform/node/SrtpTransformerNode.kt

```diff
@@ -38,9 +39,11 @@ class SrtpTransformerNode(name: String) : MultipleOutputTransformerNode(name) {
     private fun transformList(packetInfos: List<PacketInfo>): List<PacketInfo> {
         val transformedPackets = mutableListOf<PacketInfo>()
         packetInfos.forEach { packetInfo ->
-            if (transformer?.transform(packetInfo) == true) {
+            val err = transformer!!.transform(packetInfo)
```

Comment:
What's up with the `!!`? I don't think we can rely on `transformer` not being null here (hence it's nullability).  Does this not work?
```
if (transformer?.transform(packetInfo) == SrtpErrorStatus.OK) {
  ..
} else {
  ..
}
```
?

Reply:
This function is only called from within the `transformer?.let` block in `transform`.

The problem is that I need to capture the return value of `transformer.transform`.  If I do `val err = transformer?.transform(packetInfo)`, the inferred type of `val` is `SrtpErrorStatus?` which then means I need to handle the `null` case.

Reply:
Oh, if it's in the block then we should use it as `it` (or name the arg)

Reply:
Oh, I see it's not directly in the block. I think it'd be better to pass it in then or something.

Reply:
Could also do something like:
```
transformer?.transform(packetInfo)?.let {
  // 'it' here is SrtpErrorStatus
} ?: run {
  // What to do, if anything, when transformer is null
}
```

Reply:
Let me know what you think of the latest push.